src/equations/equations.jl

```diff
@@ -506,13 +505,20 @@ function waterheight end
 
 Return the product of the [`waterheight`](@ref) and the [`pressure`](@ref)
 associated to the conserved variables `u` for a given set of
-`equations`, e.g., the [`ShallowWaterEquations2D`](@ref).
+`equations`.
 
 `u` is a vector of the conserved variables at a single node, i.e., a vector
 of the correct length `nvariables(equations)`.
 """
 function waterheight_pressure end
 
+"""
+    lake_at_rest_error(u, equations)
+    
+Calculate the point-wise error for the lake-at-rest steady state solution.
+"""
```

Comment:
Does it make sense to keep the docstring?

Reply:
We may want to take the same generic approach as the other shared exported functions.